lib/hammer_cli_foreman/host.rb

```diff
@@ -176,8 +176,9 @@ class PuppetRunCommand < HammerCLIForeman::SingleResourceCommand
       resource :puppet_hosts
       action :puppetrun
 
-      def print_data(records)
-        print_message _('Puppet run triggered')
+      def execute
+        warn _('The puppetrun feature has been removed, however you can use the Remote Execution Plugin to run Puppet commands')
+        HammerCLI::EX_OK
```

Comment:
Thanks, @domitea! After some thoughts I think it would be better to fail this command in case if the command is being used within a script, so users won't be confused with OK status while nothing actually happened. The exit code should be probably `HammerCLI::EX_SOFTWARE`, the one that is used in case of server's 501 error.